app/javascript/stylesheets/layouts/sidebar.scss

```diff
@@ -37,3 +37,8 @@ $sidebar-width: 72px;
   border-bottom: thin solid theme("colors.gray.300");
   padding: 8px 12px;
 }
+
+.due-date-switch {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
1. We can remove this since it's better to use tailwind classnames in such situations.
2. I get that this is just a small SCSS change, however, please resort to the convention of writing SCSS in the correct file according to the 7:1 structure. 
Please refer to this [document](https://sass-guidelin.es/#the-7-1-pattern) for more details on it.